### PR TITLE
chore(RHTAPWATCH-539): Run yamllint checks for each opened PR

### PR DESCRIPTION
**Pull request by Kousalya1998**
- Add yaml lint in the makefile
- Add yaml lint to tekton 

Ticket:
[RHTAPWATCH-539](https://issues.redhat.com/browse/RHTAPWATCH-539)

### CODE REVIEW

File: Pipfile
Comment (yftacherzog) [inline Pipfile:9]: add newline

**Comment by yftacherzog** [comment]
nice work!

**Comment by avi-biton** [comment]
LGTM, other than the new line in `Pipfile`

File: config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml
Comment (Omeramsc) [inline config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml:33]: Is this relevant for this patch?

File: CONTRIBUTING.md
Comment (Kousalya1998) [reply CONTRIBUTING.md:18]: Will check and update the version

File: config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml
Comment (Omeramsc) [inline config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml:33]: I'll recommend adding YAML fixes to a separate commit (same PR is ok)

File: test/promql/tests/data_plane/component_errors_test.yaml
Comment (Omeramsc) [inline test/promql/tests/data_plane/component_errors_test.yaml:82]: why was the comment moved? is it relevant for this PR?

File: config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml
Comment (Kousalya1998) [reply config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml:33]: guess this time its okay..

File: test/promql/tests/data_plane/component_errors_test.yaml
Comment (Kousalya1998) [reply test/promql/tests/data_plane/component_errors_test.yaml:82]: Yes, it has some issue with that comment when I ran yamllint. so I had to move it either above or below.

**Review by hmariset — APPROVED** [review]
LGTM

**Comment by yftacherzog** [comment]
@Kousalya1998 it seems as you had some open discussions with @Omeramsc. The thing to do, next time, before merging is to get the confirmation from such reviewers that it's ok to proceed.

**Comment by Kousalya1998** [comment]
> @Kousalya1998 it seems as you had some open discussions with @Omeramsc. The thing to do, next time, before merging is to get the confirmation from such reviewers that it's ok to proceed.

@yftacherzog Thanks for letting me know. I closed it because I saw LGTM from Homaja and Avi. Sure, will wait until I get LGTM from all the reviewers and then merge.

**Comment by yftacherzog** [comment]
> > @Kousalya1998 it seems as you had some open discussions with @Omeramsc. The thing to do, next time, before merging is to get the confirmation from such reviewers that it's ok to proceed.
> 
> @yftacherzog Thanks for letting me know. I closed it because I saw LGTM from Homaja and Avi. Sure, will wait until I get LGTM from all the reviewers and then merge. 

We don't have to get lgtm from all reviewers but if someone already raised some issues then those need to be confirmed to be addressed.

**Pull request merged**
